doc/nrf/known_issues.rst

```diff
@@ -2766,6 +2923,41 @@ DRGN-18651: Scanner may fail to deliver advertising reports
   There is a chance of failure if an advertising report is received just before the scanner times out or is disabled.
   This can lead to the scanner not delivering any further advertising reports when re-enabled.
 
+.. rst-class:: v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
+
+DRGN-18714: Assert when advertising or initiating after disconnection
+  The controller could assert when starting a connectable advertiser or creating a connection too quickly after disconnection.
+
+.. rst-class:: v2-3-0 v2-2-0
+
+DRGN-18775: Periodic Advertisement Sync Transfer receiver fails to synchronize with large periodic advertising intervals
+  The Periodic Advertisement Sync Transfer (PAST) sender may generate an incorrect SyncInfo field for periodic advertising intervals greater than five seconds.
+
+.. rst-class:: v2-3-0 v2-2-0
+
+DRGN-18833: Assert when the periodic sync is in progress
+  The Periodic Advertisement Sync Transfer (PAST) sender could assert if the associated periodic sync was not fully established.
+
+.. rst-class:: v2-3-0 v2-2-0
+
+DRGN-18971: Low TX power on the nRF21540 DK in connected state
+  TX power value goes to the default one on the nRF21540 DK in connected state when using MPSL FEM and manually configuring the radio power.
```

Comment:
This issue should probably have "**Affected platforms:** nRF21540" added, TBC with @laxiLang .